src/textOnlyPackages/src/microsoft.extensions.commandlineutils.sources/5.0.1-rtm.21063.7/microsoft.extensions.commandlineutils.sources.csproj

```diff
@@ -0,0 +1 @@
+<Project Sdk="Microsoft.NET.Sdk" />
```

Comment:
The project name should have the version number in it in case we need to differentiate it from other versions (not likely, but it is the pattern).

Reply:
Fixed